server/events/vcs/bitbucketcloud/client.go

```diff
@@ -10,7 +10,7 @@ import (
 
 	"github.com/pkg/errors"
 	"github.com/runatlantis/atlantis/server/events/models"
-	validator "gopkg.in/go-playground/validator.v9"
+	"gopkg.in/go-playground/validator.v9"
```

Comment:
Why did you remove the `validator` alias? It is used several times.

Reply:
Redundant alias, removed by the IDE